public/js/home.js

```diff
@@ -131,5 +131,23 @@ $.get('/conversation/search', {
 				$contactsContent.append('<span class="recent-name"><a href="#user-modal" class="recent-contact" data-id="'+ uniqueNames[i].dataset.id +'" rel="leanModal">'+ uniqueNames[i].innerHTML +'</a></span>');
 			}  
 		}
+
+		// Displaying convoshapes
+		var $convoEntry = $('.convo');
+		$convoEntry.each(function(){
+			var $numOfNames = $(this).children('.name').length;
```

Comment:
I have a small nit: usually I see variable names with the '$' sign if they refer to a jQuery object, such as the `$convoEntry` above.  It looks like `$numOfNames` is an numeric value here.  Do you think `numOfNames` makes more sense?